lib/EnsEMBL/REST/Model/Lookup.pm

```diff
@@ -24,6 +24,7 @@ use namespace::autoclean;
 use Try::Tiny;
 use Catalyst::Exception;
 use Scalar::Util qw/weaken/;
+use Bio::EnsEMBL::DBSQL::AttributeAdaptor;
```

Comment:
Not required unless you instantiate your own AttributeAdaptor. get_adaptor takes care of the import for you